nano/node/active_transactions.cpp

```diff
@@ -623,9 +623,11 @@ void nano::active_transactions::update_active_difficulty (std::unique_lock<std::
 	{
 		std::vector<uint64_t> active_root_difficulties;
 		active_root_difficulties.reserve (roots.size ());
+		auto min_election_time (std::chrono::milliseconds (node.network_params.network.is_test_network () ? 0 : 2000));
```

Comment:
I think we should wait the amount in nano::active_transactions::request_loop, basically the average of any blocks that are stuck around since the last cycle.